aviatrix/resource_aviatrix_transit_gateway.go

```diff
@@ -1142,6 +1147,11 @@ func resourceAviatrixTransitGatewayRead(d *schema.ResourceData, meta interface{}
 		d.Set("ha_cloud_instance_id", haGw.CloudnGatewayInstID)
 		d.Set("ha_gw_name", haGw.GwName)
 		d.Set("ha_private_ip", haGw.PrivateIP)
+		lanCidr, err := client.GetTransitGatewayLanCidr(haGw.GwName)
+		if err != nil && err != goaviatrix.ErrNotFound {
+			log.Printf("[WARN] Error getting lan cidr for HA transit gateway %s due to %s", haGw.GwName, err)
+		}
+		d.Set("ha_lan_interface_cidr", lanCidr)
```

Comment:
What if err == goaviatrix.ErrNotFound? lanCidr == null?

Reply:
It’s a string, so it would be “”